### PR TITLE
chore: add text to bottom tab icons

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-871] 

### Description

This PR adds text to the bottom tabs. This comes as an attempt to make our tabs more clear to users

Group 17

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- add text to bottom tab icons - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-871]: https://artsyproduct.atlassian.net/browse/ONYX-871?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/BottomTabs/BottomTabsIcon.tsx [hunk 1]
```diff
@@ -2,11 +2,8 @@ import { useColor } from "@artsy/palette-mobile"
 import Svg, { Path } from "react-native-svg"
 import { BottomTabType } from "./BottomTabType"
 
-export const ICON_WIDTH = 56
-export const ICON_HEIGHT = 52
-
-// need to bring the icons up a smidge because of the way they were exported
-const yOffset = 2
+export const ICON_WIDTH = 53
+export const ICON_HEIGHT = 49
```
Comment (MounirDhahri) [inline src/app/Scenes/BottomTabs/BottomTabsIcon.tsx:6]: these are the new values from design

File: src/app/Scenes/BottomTabs/BottomTabsIcon.tsx [hunk 2]
```diff
@@ -18,7 +15,7 @@ export const BottomTabsIcon: React.FC<{ tab: BottomTabType; state: "active" | "i
     <Svg
       width={ICON_WIDTH}
       height={ICON_HEIGHT}
-      viewBox={`0 ${yOffset} ${ICON_WIDTH} ${ICON_HEIGHT + yOffset}`}
+      viewBox={`0 0 ${ICON_WIDTH} ${ICON_HEIGHT}`}
```
Comment (MounirDhahri) [inline src/app/Scenes/BottomTabs/BottomTabsIcon.tsx:18]: no need for the offset anymore

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (add text to bottom tab icons - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4db325d19cbd6a4b37a91d7b04dc722c01a45e91
</p>

File: src/app/Scenes/BottomTabs/BottomTabsButton.tsx [hunk 3]
```diff
@@ -84,21 +86,28 @@ export const BottomTabsButton: React.FC<BottomTabsButtonProps> = ({
...
+                    backgroundColor: "white",
+                    width: ICON_WIDTH,
+                    height: ICON_HEIGHT,
+                  }}
+                >
+                  <BottomTabsIcon tab={tab} state="active" />
+                </Animated.View>
+              </IconWrapper>
+            </Flex>
+            <Flex height={BOTTOM_TABS_TEXT_HEIGHT}>
```
Comment (olerichter00) [inline src/app/Scenes/BottomTabs/BottomTabsButton.tsx:107]: Just to make the code easier and more pleasant to read :)

```suggestion
            </Flex>

            <Flex height={BOTTOM_TABS_TEXT_HEIGHT}>
```

**Review by olerichter00 — DISMISSED** [review]
I like this update a lot! 🌟

**Review by gkartalis — DISMISSED** [review]
👏

**Pull request merged**